src/routes/docs/products/network/cdn/+page.markdoc

```diff
@@ -0,0 +1,25 @@
+---
+layout: article
+title: Content delivery network (CDN)
```

Comment:
Content Delivery Network may be better kept as title case, considering it's become a standardized term